test/functional/pdo_sqlsrv/pdostatement_GetDataType.phpt

```diff
@@ -1,305 +1,302 @@
+    $result = $stmt->fetch(PDO::FETCH_ASSOC);
+    var_dump($result);
 }
-	
 
-try
+function testXml($db, $tbname)
 {
-	$db = connect();
-	//$sql = "INSERT INTO PDO_AllTypes(BigIntCol,BitCol,IntCol,) VALUES(" . GetSampleData(4) . ",1,)";
-	//$numRows = $db->exec($sql);
-	
-	echo "Test_1 : bigint data type :\n";
-	testBigInt($db);
-	echo "Test_2 : bit data type :\n";
-	testBit($db);
-	echo "Test_3 : int data type :\n";
-	testInt($db);
-	echo "Test_4 : smallint data type:\n";
-	testSmallInt($db);
-	echo "Test_5 : tinyint data type:\n";
-	testTinyInt($db);
-	echo "Test_6 : decimal data type:\n";
-	testDecimal($db);	
-	echo "Test_7 : numeric data type:\n";
-	testNumeric($db);  
-	echo "Test_8 : money data type:\n";
-	testMoney($db);
-	echo "Test_9 : smallmoney data type:\n";
-	testSmallMoney($db);	
-	echo "Test_10 : float data type:\n";
-	testFloat($db);
-	echo "Test_11 : real data type:\n";
-	testReal($db);
-	echo "Test_12 : char data type:\n";
-	testChar($db);
-	echo "Test_13 : varchar data type:\n";
-	testVarchar($db);
-	echo "Test_14 : text data type:\n";
-	testText($db);
-	echo "Test_15 : nchar data type:\n";
-	testNChar($db);
-	echo "Test_16 : nvarchar data type:\n";
-	testNVarchar($db);	
-	echo "Test_17 : image data type:\n";
-	testImage($db);
-	echo "Test_18 : binary data type:\n";
-	testBinary($db);
-	echo "Test_19 : varbinary data type:\n";
-	testVarbinary($db);
-	echo "Test_20 : smalldatetime data type:\n";
-	testsmalldatetime($db);
-	echo "Test_21 : datetime data type:\n";
-	testDateTime($db);
-	echo "Test_22 : datetime2 data type:\n";
-	testsmalldatetime($db);
-	echo "Test_23 : datetimeoffset data type:\n";
-	testDatetimeoffset($db);
-	echo "Test_24 : time data type:\n";
-	testTime($db);
-	echo "Test_25 : Uniqueidentifier data type:\n";
-	testUniqueidentifier($db);
-	echo "Test_26 : VarbinaryMax data type:\n";
-	testVarbinaryMax($db);
-	echo "Test_27 : VarcharMax data type:\n";
-	testVarcharMax($db);
-	echo "Test_28 : xml data type:\n";
-	testXml($db);
-	echo "Test_29 : ntext data type:\n";
-	testNText($db);
-	echo "Test_30 : nvarcharmax data type:\n";
-	testNVarcharMax($db);
-	echo "Test_31 : date data type:\n";
-	testDate($db);
-}
-catch (PDOException $e)
-{
-	var_dump($e);
+    $stmt = $db->query("SELECT XmlCol FROM $tbname");
+    $result = $stmt->fetch(PDO::FETCH_ASSOC);
+    var_dump($result);
 }
 
+try {
+    $db = connect();
+    $tbname = "PDO_AllTypes";
+    createAndInsertTableAllTypes($db, $tbname);
+
+    echo "Test_1 : bigint data type :\n";
+    testBigInt($db, $tbname);
+    echo "Test_2 : bit data type :\n";
+    testBit($db, $tbname);
+    echo "Test_3 : int data type :\n";
+    testInt($db, $tbname);
+    echo "Test_4 : smallint data type:\n";
+    testSmallInt($db, $tbname);
+    echo "Test_5 : tinyint data type:\n";
+    testTinyInt($db, $tbname);
+    echo "Test_6 : decimal data type:\n";
+    testDecimal($db, $tbname);
+    echo "Test_7 : numeric data type:\n";
+    testNumeric($db, $tbname);
```

Comment:
Please use for loops and array to further simply this large block of very similar lines